_includes/performance.html

```diff
@@ -0,0 +1,64 @@
+<div class="full-width-bg component">
+  <div class="grid-wrapper">
+    <div class="width-3-12 width-12-12-m img-vert-center">
+      <img class="light-only" src="{{site.baseurl}}/assets/images/performance/icon-performance.svg" alt="Performance icon">
+      <img class="dark-only" src="{{site.baseurl}}/assets/images/performance/icon-performance-dark.svg" alt="Performance icon">
+    </div>
+    <div class="width-9-12 width-12-12-m">
+      <h2>The Quarkus Way</h2>
```

Comment:
HTML makes reviewing very hard. Could you try the technique I used for continuum?